subprojects/core/src/main/java/org/gradle/api/internal/changedetection/rules/TaskTypeTaskStateChanges.java

```diff
@@ -46,7 +46,7 @@ public boolean accept(TaskStateChangeVisitor visitor) {
             return visitor.visitChange(new DescriptiveChange("Task '%s' %s", task.getIdentityPath(), taskImplementation.getUnknownReason()));
         }
         if (prevImplementation.isUnknown()) {
-            return visitor.visitChange(new DescriptiveChange("Task '%s' %s during the previous execution", task.getIdentityPath(), prevImplementation.getUnknownReason()));
+            return visitor.visitChange(new DescriptiveChange("During the previous execution task '%s' %s", task.getIdentityPath(), prevImplementation.getUnknownReason()));
```

Comment:
> During the previous execution task

..." of the build" maybe?

Reply:
I reworded it to `During the previous execution of the task ':test', it was loaded with an unknown classloader`.

Reply:
I reworded to `During the previous execution of the task ':test', it was loaded with an unknown classloader`, WDYT?